_observing-your-data/alerting/security.md

```diff
@@ -40,6 +40,12 @@ Once a monitor is created, the Alerting plugin will continue executing the monit
 
 If your monitor's trigger has notifications configured, the Alerting plugin continues to send out notifications regardless of destination type. To stop notifications, a user must manually delete them in the trigger's actions.
 
+### A note on alerts and fine-grained access control
+
+When a trigger generates an alert, the monitor configuration, the alert itself, and any notification that is sent to a channel may include metadata describing the index being queried. By design, the plugin must extract data and store it as metadata outside of the index. OpenSearch Security’s document-level security (DLS) and field-level security (FLS) fine-grained access controls are designed to protect data in the index. But once data is stored outside the index as metadata, users with access to monitor configurations, alerts, and their notifications will be able to view this metadata and possibly infer the contents and quality of data in the index, which would otherwise be concealed by DLS and FLS access control.
```

Comment:
Same edits as above. In case you need to add the 'fine-grained' terminology in there, feel free to add back. It just makes the sentence a little clunky. You could flip it: In OpenSearch, the fine-grained access controls for DLS and FLS are designed to..."